rest/data/admin.go

```diff
@@ -0,0 +1,104 @@
+package data
+
+import (
+	"github.com/evergreen-ci/evergreen/model/admin"
+	"github.com/evergreen-ci/evergreen/model/event"
+)
+
+type DBAdminConnector struct{}
+
+// GetAdminSettings retrieves the admin settings document from the DB
+func (ac *DBAdminConnector) GetAdminSettings() (*admin.AdminSettings, error) {
+	return admin.GetSettings()
+}
+
+// SetAdminSettings sets the admin settings document in the DB and event logs it
+func (ac *DBAdminConnector) SetAdminSettings(settings admin.AdminSettings) error {
```

Comment:
It seems odd that this method and the next two just delegate to other functions in this same package. It seems like we can eliminate a bunch of code and as a result do less jumping around if we move the logic from UpdateSettings(), UpdateBanner(), and UpdateServiceFlags() into these methods.

Reply:
Done